app/views/chugs/_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<div>
+  <%= form_for @chug do |f| %>
+    <% if @chug.errors.any? %>
+        <div id="error_explanation">
+          <h2><%= pluralize(@chug.errors.count, 'error') %> prohibited this chug from being saved:</h2>
+          <ul>
+            <% @chug.errors.full_messages.each do |msg| %>
+                <li><%= msg %></li>
+            <% end %>
+          </ul>
+        </div>
+    <% end %>
```

Comment:
Met de `action_object` helpers die je nu gebruikt is dit blok nooit gebuikt dus kun je het om mij weggooien.